cmd/scheduler/cmd/scheduler.go

```diff
@@ -147,7 +147,30 @@ func Run(ctx context.Context, options *scheduleroptions.Options) error {
 	}
 	edgeSharedInformerFactory := edgeinformers.NewSharedInformerFactoryWithOptions(edgeViewClusterClientset, resyncPeriod)
 
-	// create the edge-scheduler
+	// create workloadSharedInformerFactory
+	// solely for SyncTarget because "kcpSharedInformerFactory.Workload().V1alpha1().SyncTargets()" doens't work (why?)
+	workloadConfig, _ := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		clientcmd.NewDefaultClientConfigLoadingRules(),
+		&clientcmd.ConfigOverrides{
+			Context: clientcmdapi.Context{
+				Cluster:  "root",
+				AuthInfo: "kcp-admin",
+			},
+		},
+	).ClientConfig()
+	workloadViewConfig, err := configForViewOfExport(ctx, workloadConfig, "workload.kcp.io")
```

Comment:
BTW, `edgeclient.ConfigForScheduling` used above is a hardcoded version of `configForViewOfExport`.  I think it is better to consistently use the less hardcoded version.

Reply:
Agree, fixed by https://github.com/kcp-dev/edge-mc/pull/206/commits/aff3b307a2c7c042d5b2253341551972764158cc.